src/pages/_app.tsx

```diff
@@ -3,11 +3,13 @@ import "~/globals.css";
 import {AppProps} from "next/app";
 import Head from "next/head";
 
+import {PAGES} from "~/utils/pages";
+
 const website = ({Component, pageProps}: AppProps) => {
   return (
     <>
       <Head>
-        <title>PAGES.HOME.title</title>
+        <title>{PAGES.HOME.title}</title>
```

Comment:
This makes the same title from home page even for blog pages. The biggest problem was with blog.
The <title> here is completely useless, because page decides on its title, not the whole app.